mailhandler/src/main/java/org/eclipse/angus/mail/util/logging/LogManagerProperties.java

```diff
@@ -65,6 +66,8 @@
  */
 final class LogManagerProperties extends Properties {
 
+    private final ReentrantLock lock = new ReentrantLock();
```

Comment:
I think you can drop the lock and the synchronized keywords for this class.  A lot of that was designed to piggyback on properties using synchronized but that is no longer the case.

Reply:
Keeping lock is fine for compatibility.